### PR TITLE
feat(java): Implement global timeout configuration for generated clients.

### PR DESCRIPTION
**Pull request by eyw520**
## Description
Support timeout configuration in Java client builder.

## Changes Made
- Edit `ClientOptionsGenerator` to support timeout method in generated ClientOptions.
- Edit `RootClientGenerator` to support timeout method during ClientOptions build.

## Testing
- [x] Unit tests added/updated
- [x] Manual testing completed

### CODE REVIEW

File: generators/java/sdk/src/main/java/com/fern/java/client/generators/RootClientGenerator.java [hunk 1]
```diff
@@ -201,6 +201,14 @@ private TypeSpec getClientBuilder() {
                     .build());
         }
 
+        clientBuilder.addMethod(MethodSpec.methodBuilder("timeout")
+            .addModifiers(Modifier.PUBLIC)
+            .addParameter(int.class, "timeout")
+            .returns(builderName)
+            .addStatement("this.$L.timeout(timeout)", CLIENT_OPTIONS_BUILDER_NAME)
+            .addStatement("return this")
+            .build());
```
Comment (dsinghvi) [inline generators/java/sdk/src/main/java/com/fern/java/client/generators/RootClientGenerator.java:210]: nit: can we add a docstring to this method that this sets the timeout in seconds

**Comment by dsinghvi** [comment]
Once you regenerate the seed snapshots, itll be easy to verify but this looks right to me!

File: generators/java/sdk/src/main/java/com/fern/java/client/generators/RootClientGenerator.java (hunk 1 above)
Comment (eyw520) [reply generators/java/sdk/src/main/java/com/fern/java/client/generators/RootClientGenerator.java]: Done

File: seed/java-sdk/alias/src/main/java/com/seed/alias/core/ClientOptions.java [hunk 2]
```diff
@@ -78,6 +82,8 @@ public static final class Builder {
 
         private final Map<String, Supplier<String>> headerSuppliers = new HashMap<>();
 
+        private int timeout = 0;
```
Comment (dsinghvi) [inline seed/java-sdk/alias/src/main/java/com/seed/alias/core/ClientOptions.java:85]: can we initialize the timeout to be 60 seconds

File: seed/java-sdk/alias/src/main/java/com/seed/alias/core/ClientOptions.java [hunk 3]
```diff
@@ -93,11 +99,17 @@ public Builder addHeader(String key, Supplier<String> value) {
             return this;
         }
 
+        public Builder timeout(int timeout) {
```
Comment (dsinghvi) [inline seed/java-sdk/alias/src/main/java/com/seed/alias/core/ClientOptions.java:102]: nit: there should be a javadoc above this method that says `Override the timeout in seconds. Defaults to 60 seconds.`

File: generators/java/sdk/versions.yml [hunk 4]
```diff
@@ -1,3 +1,10 @@
+- changelogEntry:
+    - summary: |
+        Implement global timeout configuration for generated clients.
```
Comment (dsinghvi) [inline generators/java/sdk/versions.yml:3]: nit: can you add a code snippet of how someone would override the timeout: 

```java
var client = Client.builder()
  .apiKey("...")
  .timeout(40)
  .build();
```

**Review by dsinghvi — APPROVED** [review]
Overall looks great, approved with some tiny comments

File: generators/java/sdk/versions.yml (hunk 4 above)
Comment (eyw520) [reply generators/java/sdk/versions.yml:3]: Done

File: seed/java-sdk/alias/src/main/java/com/seed/alias/core/ClientOptions.java (hunk 3 above)
Comment (eyw520) [reply seed/java-sdk/alias/src/main/java/com/seed/alias/core/ClientOptions.java:102]: Done.

File: seed/java-sdk/alias/src/main/java/com/seed/alias/core/ClientOptions.java (hunk 2 above)
Comment (eyw520) [reply seed/java-sdk/alias/src/main/java/com/seed/alias/core/ClientOptions.java:85]: Yep; done.

**Pull request merged**